### DISCUSSION TITLE
Don't hide quotes by default in JSON files

### DISCUSSION THREAD

**Discussion by akoskm** [LazyVim/LazyVim#2390] [Q&A] [answered]
Hi 👋 

Which plugin controls showing the quotes in JSON files by default? A couple of days ago, I thought of a JSON file that is incorrect, only to figure out later that only the quotes weren't showing in the editor:

![image](https://github.com/LazyVim/LazyVim/assets/3111627/5e4555a0-673c-4991-ba1e-008cf0c5b0bd)

**Comment by folke** [comment, marked as answer]
https://github.com/LazyVim/LazyVim/commit/23fe52acd78c8e8f7c35c3ce4c1bae830adfa357

**Reply by akoskm** [reply]
I saw the term conceal earlier in the UI, but I had no idea what it meant. Thanks 😃
